.github/contributing.md

```diff
@@ -0,0 +1,52 @@
+# Contributors Guide
+
+Interested in contributing? Awesome! Before you do though, please read our
+[Code of Conduct](https://slackhq.github.io/code-of-conduct). We take it very seriously, and expect that you will as
+well.
+
+There are many ways you can contribute! :heart:
+
+## Bug Reports and Fixes :bug
+
+- If you find a bug, please search for it in the [Issues](https://github.com/slackapi/manifest-schema/issues), and if it isn't already tracked,
+   [create a new issue](https://github.com/slackapi/manifest-schema/issues/new). Fill out the "Bug Report" section of the issue template. Even if an Issue is closed, feel free to comment and add details, it will still
+   be reviewed.
+- Issues that have already been identified as a bug (note: able to reproduce) will be labelled `bug`.
+- If you'd like to submit a fix for a bug, [follow the maintainers guide](./maintainers_guide.md) and mention the Issue number.
+- Include tests that isolate the bug and verifies that it was fixed.
+
+## New Features :bulb
+
+- If you'd like to add new functionality to this project, describe the problem you want to solve in a [new Issue](https://github.com/slackapi/manifest-schema/issues/new).
+- Issues that have been identified as a feature request will be labelled `enhancement`.
+- If you'd like to implement the new feature, please wait for feedback from the project
+   maintainers before spending too much time writing the code. In some cases, `enhancement`s may
+   not align well with the project objectives at the time.
+
+## Tests :mag:, Documentation :books:, Miscellaneous :sparkles
+
+- If you'd like to improve the tests, you want to make the documentation clearer, you have an
+   alternative implementation of something that may have advantages over the way its currently
+   done, or you have any other change, we would be happy to hear about it!
+- If its a trivial change, go ahead and [follow the maintainers guide](./maintainers_guide.md) with the changes you have in mind.
+- If not, [open an Issue](https://github.com/slackapi/manifest-schema/issues/new) to discuss the idea first.
```

Comment:
Should there be an issue template in this repo? Similar to this for specific issue types: https://github.com/slackapi/bolt-python/tree/main/.github/ISSUE_TEMPLATE or something just as a catch-all for all issues: https://github.com/slackapi/bolt-python/blob/main/.github/issue_template.md

Reply:
Ahh yess I think I forgot to add it, will add a catch-all template 😎